src/Command/UpdateCdnStatusCommand.php

```diff
@@ -84,6 +85,9 @@ public function execute(InputInterface $input, OutputInterface $output)
         $this->output = $output;
 
         $provider = $this->getProvider();
+
+        \assert($provider instanceof BaseProvider);
```

Comment:
What the purpose of this assert ?

I prefer to understand because it often show an issue
- with our return type
- with the use of the return values
- with our interfaces

Reply:
I need to address #1824 in order to fix this comment.

Reply:
Can we have a comment with the ref of the issue then ? (just in case)

Reply:
I finally added the `@method` annotation at `CDNIterface`.